website/app/initializers/showdown-extensions.js

```diff
@@ -92,7 +93,11 @@ export function initialize(/* application */) {
           language = 'shell';
         }
 
-        let highlightedCodeBlock = Prism.highlight(codeblock, Prism.languages[language], language) + end;
+        if (Prism.languages[language]) {
```

Comment:
maybe add a comment here explaining the difference between the two logical paths? (not obvious)

Reply:
Added comments in https://github.com/hashicorp/design-system/pull/1749/commits/a611089997a3b59117754f72a32f93b37c4285bf